src/GitHub.Exports.Reactive/ViewModels/IPullRequestListViewModel.cs

```diff
@@ -4,39 +4,33 @@
 using System.Windows.Input;
 using ReactiveUI;
 using System.Collections.ObjectModel;
+using GitHub.Settings;
 
 namespace GitHub.ViewModels
 {
-    public class PullRequestState
+    public class PullRequestState : ViewModelItemContainer
     {
-        public PullRequestState()
-        {
-        }
-
-        public PullRequestState(bool isOpen, string name)
-        {
-            IsOpen = isOpen;
-            Name = name;
-        }
-
         public bool? IsOpen;
-        public string Name;
-        public override string ToString()
-        {
-            return Name;
-        }
+    }
+
+    public class PullRequestSortOrder : ViewModelItemContainer
+    {
+        public IComparer<IPullRequestModel> Comparer;
+        public SortOrder SortOrder;
```

Comment:
These should probably be properties. Most likely they should be `readonly` properties. I know that makes creating instances of these not look as nice as using property initializers when constructing an instance, but if these properties should never be mutated after construction, then it guides future consumers to the right behavior.

However, if you're two-way binding these properties to a UI, then ignore the `readonly` part and make them properties. But in that case they should probably be reactive properties (or implement INPC).